lib/alexaSmartHomeV2.js

```diff
@@ -105,10 +105,20 @@ function AlexaSH2(adapter) {
                         } else if (lang === 'ru') {
                             text = 'Состояние ' + obj.friendlyName + ' <emphasis>установлено</emphasis> на ' + value;
                         } else {
-                            text = obj.friendlyName + ' set to ' + value + ' <emphasis>percent</emphasis>';
+                            text = obj.friendlyName + ' <emphasis>set to</emphasis> ' + value + ' percent';
                         }
                         break;
 
+                    case 'K':
+                        if (lang === 'de') {
+                            text = obj.friendlyName + ' wird auf ' + value + ' Kelvin <emphasis>gesetzt</emphasis>';
+                        } else if (lang === 'ru') {
+                            text = 'Состояние ' + obj.friendlyName + ' <emphasis>установлено</emphasis> на ' + value;

```

Comment:
This is the line from `percent` but I can't read russian :D

Reply:
I'm a bit unsure what all the text messages should do anyway. The Alexa Smart Home Skill at least in v2 (I'm not sure about v3) can't return any text (maybe for Locks and Temperature) and Alexa always answers with "Ok" (except if it fails). As far as I can tell the only thing the writeResponse thing does is write that text to a state. But I have not yet figured out what that state is good for...

Reply:
I think that's only for ioBroker if the user wants to evaluate what Alexa/ioBroker.cloud did, e.g. for visualization.